Sources/Mistica/Components/Button/Button.swift

```diff
@@ -56,6 +63,29 @@ open class Button: UIControl {
         }
     }
 
+    public enum RightImage {
+        case chevron
+        case custom(image: UIImage)
+
+        private var _image: UIImage {
+            switch self {
+            case .chevron: return .chevron
+            case let .custom(image: image): return image
+            }
+        }
+
+        var image: UIImage {
+            _image.withRenderingMode(.alwaysTemplate)
+        }
```

Comment:
Why have `image: UIImage` and `_image: UIImage`?

```suggestion
        var image: UIImage {
            switch self {        
            case .chevron: 
                return .chevron.withRenderingMode(.alwaysTemplate)
            case let .custom(image: image):
                return image.withRenderingMode(.alwaysTemplate)
            }                               
        }
```

Reply:
The point is that the image should have the same color as the title, so it should be rendered as a template to be tinted. The idea is that I didn't want to set all the image cases with `.withRenderingMode(.alwaysTemplate)`. The private `_image` is the image itself and the published property `image` is which sets the property